sunscreen_tfhe/src/params.rs

```diff
@@ -223,6 +223,15 @@ pub const LWE_512_128: LweDef = LweDef {
     std: Stddev(0.0004899836456140595),
 };
 
+/// 128-bit secure parameters for a GLWE instance with a dimension of 512.
+pub const GLWE_1_512_128: GlweDef = GlweDef {
+    dim: GlweDimension {
+        size: GlweSize(1),
+        polynomial_degree: PolynomialDegree(512),
+    },
+    std: Stddev(0.0004899836456140595),
+};
+
```

Comment:
Add a test for this in `check_security_levels` so that if we end up changing the security level models we will know which parameters need changing.